submit/util.py

```diff
@@ -7,13 +7,14 @@
 
 from arxiv.base import logging
 from arxiv.base.globals import get_application_global
+from arxiv.submission.services.classic.exceptions import Unavailable
 import arxiv.submission as events
 
 logger = logging.getLogger(__name__)
 logger.propagate = False
 
 
-@retry(tries=3, delay=0.25, backoff=3)
```

Comment:
The changes to this line don't seem related to this bug. Do they belong in a different PR or are the trivial or are they part of some debugging work?

Reply:
Yeah, was just opportunistic tuning